spec/jobs/rapid_ready_for_decision/disability_compensation_job_spec.rb

```diff
@@ -174,10 +194,23 @@
       end
 
       it 'raises an ArgumentError' do
-        Sidekiq::Testing.inline! do
-          expect do
+        VCR.use_cassette('evss/claims/claims_without_open_compensation_claims') do
+          Sidekiq::Testing.inline! do
+            expect do
+              subject.perform_async(submission.id)
+            end.to raise_error(ArgumentError, 'no ICN passed in for LH API request.')
+          end
+        end
+      end
+    end
+
+    context 'when there are pending claims that cause EP 400 errors' do
+      it 'off ramps onto the regular process' do
+        VCR.use_cassette('evss/claims/claims') do
+          Sidekiq::Testing.inline! do
+            expect(RapidReadyForDecision::HypertensionMedicationRequestData).not_to receive(:new)
```

Comment:
How about checking at a higher/earlier level:
```suggestion
            expect(Lighthouse::VeteransHealth::Client).not_to receive(:new)
```

In addition, you could check that `pending_eps?(form526_submission.auth_headers)` returns true.